pystruct/learners/n_slack_ssvm.py

```diff
@@ -12,13 +12,17 @@
 import cvxopt
 import cvxopt.solvers
 
-from sklearn.externals.joblib import Parallel, delayed
+from sklearn.externals.joblib import Parallel, delayed, cpu_count
 from sklearn.utils import gen_even_slices
 
 from .ssvm import BaseSSVM
 from ..utils import unwrap_pairwise, find_constraint
 
 
+def find_constraint_map(args):
+    return find_constraint(* args)
```

Comment:
Ok, then it's probably a flaw in my logic somewhere. I'll work in all we've talked about here and see if I can get rid of the defaults to Parallel and still pass those tests.